linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```diff
@@ -1,5 +1,7 @@
 {{%- if product in ["rhel8"] %}}
     {{%- set kmod_audit="-a always,exit -F path=/usr/bin/kmod -F perm=x -F auid>=1000 -F auid!=unset -F key=privileged" %}}
+{{%- elif product in ["rhel7"] %}}
+    {{%- set kmod_audit="-w /usr/bin/kmod -p x -F auid!=unset -k module-change" %}}
```

Comment:
@ggbecker I just looked it again and you're right re: remediation.

Looking over the template, I could also just add 3 variables (that default to existing behavior) to the template to set behavior accordingly. As a bonus, if we run into a similar situation (like a rule wants `-w` or not having `-a always,exit`, we can account for it and we can set them per product (which I am working through now).

Currently looking at:
* `min_auid` - variable (defaults to `auid`) that represents the minimum auid value for an audit. This could be instead be a boolean switch, but this lets us vary a specific rule from `auid`. If it's 0, the check is omitted.
* `skip_action` - boolean value (`true/false`; defaults to `false`) that suppresses the generation of `-a always,exit` when set to `true`.
* `watch` - changes from `-F path=xxx` to `-w xxx` when `true` (defaults to false).

Reply:
If it's considered to be "too much", then we can stuff overrides into the same rule implementation for RHEL7.